src/sass/main.scss

```diff
@@ -5,10 +5,25 @@ html {
 }
 
 // used for theme skins with a default to gray
-@mixin color_cascade($color: #999) {
+@mixin color_cascade($color: #999, $alert-color: red, $warning-color: orange) {
```

Comment:
I think we are using `mixins` incorrectly here. I might try and update this if that's okay?